sentinel/src/schedule/index.js

```diff
@@ -1,28 +1,32 @@
-const async = require('async');
 const moment = require('moment');
 const config = require('../config');
 const transitionsLib = config.getTransitionsLib();
 const date = transitionsLib.date;
 const logger = require('../lib/logger');
 
 const tasks = {
-  dueTasks: transitionsLib.dueTasks,
+  dueTasks: {
+    execute: cb => {
+      if (module.exports._sendable(config, moment(date.getDate()))) {
+        transitionsLib.dueTasks.execute(cb);
+      } else {
+        cb();
+      }
+    }
+  },
   reminders: require('./reminders'),
   replications: require('./replications'),
   outbound: require('./outbound'),
   purging: require('./purging')
 };
+const ongoingTasks = new Set();
```

Comment:
Can we rename `task` to `job` (or something else) throughout this doc? `task` feels overloaded now...

Reply:
Good call! Changed to job